cmd/calyptia/delete_core_instance.go

```diff
@@ -0,0 +1,166 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/calyptia/api/types"
+	"github.com/calyptia/cli/k8s"
+	"github.com/spf13/cobra"
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func newCmdDeleteCoreInstance(config *config, testClientSet kubernetes.Interface) *cobra.Command {
+	var skipError bool
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+	configOverrides := &clientcmd.ConfigOverrides{}
+	cmd := &cobra.Command{
+		Use:               "core_instance CORE_INSTANCE",
+		Aliases:           []string{"instance", "aggregator"},
+		Short:             "Delete a core instance",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: config.completeAggregators,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+
+			aggregatorKey := args[0]
+			var agg types.Aggregator
+			var err error
+			if validUUID(aggregatorKey) {
```

Comment:
```suggestion
```
A name could have UUID format.
That's why `loadAggregatorID` accepts just a "key" that could be the name or ID and does the resolution and gives you back the ID.